src/main/json/r6OperatorCounters.json

```diff
@@ -1503,6 +1503,33 @@
                     "properties": {
                         "Description": "Dokkaebi can use her Logic Bomb to ring Caveria's phone. This will reveal Caveria's position when roaming even if Silent Step is active."
                     }
+                },
+                {
+                  "id": "143",
+                  "type": "Hard Counter",
+                  "startNode": "48".
+                  "endNode": "47",
+                  "properties": {
+                      "Description": "Nøkk's HEL will remove her image from Mozzie's drones."
```

Comment:
This counter exists on the graph. (ID 130) would you be able to remove this from your commit, please?